src/site/components/merger-social.html

```diff
@@ -40,7 +44,11 @@ <h4>{{ subs.subhead }}</h4>
                     {% endif %}
                     >
                     {% if link.icon != empty %}
-                      <i class="fa fab {{link.icon}}"></i>
+                      {% if is_social_media_links %}
+                        <i class="fab {{link.icon}}"></i>
+                      {% else %}
+                        <i class="fas {{link.icon}}"></i>
+                      {% endif %}
```

Comment:
I don't like this, but this template will be deprecated when the Drupal template comes along.